BindingContext/explainer.md

```diff
@@ -144,42 +175,60 @@ A server may also return a new BPoP nonce on any 200 response.
 ```
 HTTP/1.1 200 Ok
 BPoP-Nonce: eyJ7S_zG.eyJH0-Z.HX4w-7v
+refresh-in: 300
+expires-in: 300
 ```
 
 The client SHOULD start using the new nonce on the next request.
 
 ### Javascript
 
-For [Fetch](https://fetch.spec.whatwg.org/), the client should include a BPoP proof when *includeCredentials* is true.
+For [Fetch](https://fetch.spec.whatwg.org/), the client should include a BPoP proof when _includeCredentials_ is true.
 
 ## Privacy Considerations
 
 This proposal is intended to operate in the two modes already shipping in most majors browsers - a standard mode for broad audiences and an enterprise mode (equivalent to Chromium's "enterprise policy"). These modes should appear effectively the same from the perspective of a server which is unaware of enterprise features.
 
 In standard mode:
- - The browser uses one public private key pair per eTLD+1.
- - The browser chooses where these keys are stored (software, hardware, roaming, etc.)
- - The browser is licensed to clear these keys at any time for any reason (like cookies). The browser should avoid clearing these keys too frequently as such clears may be disruptive to the user experience (e.g. require the user to sign in again).
- - The browser SHOULD clear these keys as part of any other "delete site data" experience.
+
+- The browser uses one public private key pair per eTLD+1.
+- The browser chooses where these keys are stored (software, hardware, roaming, etc.)
+- The browser is licensed to clear these keys at any time for any reason (like cookies). The browser should avoid clearing these keys too frequently as such clears may be disruptive to the user experience (e.g. require the user to sign in again).
+- The browser SHOULD clear these keys as part of any other "delete site data" experience.
 
 If the user's machine is governed by an enterprise policy, this spec envisions that the behavior of BPoP changes as follows:
- - Instead of the browser choosing keys for BPoP proofs, the enterprise policy provider chooses the keys.
- - The enterprise policy provider may augment the BPoP proof payload with additional claims.
+
+- Instead of the browser choosing keys for BPoP proofs, the enterprise policy provider chooses the keys.
```

Comment:
Maybe let's provide a bit more information on what this means in practice to make sure we're aligned on what this means?
For example, I'm thinking that it's important that a mechanism that browsers implement should provide a way for code execution in the native process (e.g. WAM on Windows, SSO extension on Mac, broker service inside Authenticator on Android) and the native process should get sufficient information about the caller to verify it (as opposed to a static configuration driven by the enterprise policy telling browser how to generate keys). 

Another aspect is that I'm not sure about is whether we'd want that process to supply/attest keys, or generate the actual BPop proof? Would be great to align and call out as well to drive consistency in browser implementations once we get that far.